.travis.yml

```diff
@@ -2,11 +2,7 @@ language: python
 python:
   - 2.6
   - 2.7
-  - 3.3
```

Comment:
You don't have to remove these guys. Look here: https://github.com/ClearcodeHQ/mirakuru/blob/master/.travis.yml#L32

You can run tests, but exclude them from results.